### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by epiro-ps**
Closes

other changes

- [ ] The impure tests via `nix run .#offchain:test` were run and all are passing.
- [ ] Where applicable, grammar, punctuation, and spelling within code and comments should be plausibly correct.
- [ ] The issue(s) that the work is addressing should be linked in the PR description.
- [ ] Any significant changes beyond what's clear from the linked issue(s) are documented in the PR description

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2022, Ardana Labs
```
Comment (Fresheyeball) [inline LICENSE]: WHAT ABOUT US?!?! :cry:

Comment (epiro-ps) [reply LICENSE]: DONE

**Review by Geometer1729 — APPROVED** [review]
LGTM

File: LICENSE (hunk 1 above)
Comment (Fresheyeball) [reply LICENSE]: Gracias

**Pull request merged**